web/scenes/Portal/Teams/TeamId/Team/page/index.tsx

```diff
@@ -7,9 +10,30 @@ type TeamIdPageProps = {
   searchParams: Record<string, string> | null | undefined;
 };
 
-export const TeamIdPage = (props: TeamIdPageProps) => {
+export const TeamIdPage = async (props: TeamIdPageProps) => {
   const { params } = props;
   const teamId = params?.teamId as `team_${string}`;
+  const session = await getSession();
+  const user = session?.user as Auth0SessionUser["user"];
+
+  const isTeamMember = user?.hasura?.memberships?.some(
+    (membership) => membership.team?.id === teamId,
+  );
+
+  if (!isTeamMember) {
+    return (
+      <div className="flex h-full items-center justify-center gap-x-2">
+        <Typography variant={TYPOGRAPHY.M2}>401</Typography>
+
+        <div className="h-5 w-px bg-grey-400" />
+
+        <Typography variant={TYPOGRAPHY.H6} className="text-grey-400">
+          You are not a member of this team
```

Comment:
Can we just do a <error component with 401 forbidden like the other pages? 
```suggestion
          You are not a member of this team
```

Reply:
Updated

<img width="1092" alt="image" src="https://github.com/worldcoin/developer-portal/assets/89008845/53d55d75-ec02-4b91-9304-06dda3aaf809">


<img width="1574" alt="image" src="https://github.com/worldcoin/developer-portal/assets/89008845/bc56e9f8-de98-423a-bde2-e8620bb8c372">